src/pages/userPage/EditUserPage/EditUserPage.tsx

```diff
@@ -51,7 +51,7 @@ export default function () {
   const wrongOnlyLettersMessage = shouldContain("тільки літери");
   const wrongAllVariantsMessage = shouldContain("літери, символи та цифри");
   const [form] = Form.useForm();
-  const MAX_AGE = 100;
+  const MIN_AVILABLE_DATE = "01.01.1900";
```

Comment:
I guess it should be:
```suggestion
  const MIN_AVAILABLE_DATE = "01.01.1900";
```

Reply:
Thanks)

Reply:
Done!